medmodels/medrecord/querying.pyi

```diff
@@ -50,132 +72,231 @@ class EdgeOperand:
+    def modulo(self, value: AttributeArithmeticOperand) -> SingleAttributeOperand: ...
+    def power(self, value: AttributeArithmeticOperand) -> SingleAttributeOperand: ...
+    def round(self) -> SingleAttributeOperand: ...
+    def ceil(self) -> SingleAttributeOperand: ...
+    def floor(self) -> SingleAttributeOperand: ...
+    def absolute(self) -> SingleAttributeOperand: ...
+    def sqrt(self) -> SingleAttributeOperand: ...
+    def trim(self) -> SingleAttributeOperand: ...
+    def trim_start(self) -> SingleAttributeOperand: ...
+    def trim_end(self) -> SingleAttributeOperand: ...
+    def lowercase(self) -> SingleAttributeOperand: ...
+    def uppercase(self) -> SingleAttributeOperand: ...
+    def slice(self, start: int, end: int) -> SingleAttributeOperand: ...
+    def either_or(
+        self, either: SingleAttributeOperand, or_: SingleAttributeOperand
+    ) -> None: ...
 
-class EdgeValuesOperand:
-    def max(self) -> EdgeValueOperand: ...
-    def min(self) -> EdgeValueOperand: ...
-    def mean(self) -> EdgeValueOperand: ...
-    def all(self) -> EdgeValueOperand: ...
-    def any(self) -> EdgeValueOperand: ...
-    def greater_than(self, value: ComparisonOperand) -> None: ...
-    def greater_than_or_equal(self, value: ComparisonOperand) -> None: ...
-    def less_than(self, value: ComparisonOperand) -> None: ...
-    def less_than_or_equal(self, value: ComparisonOperand) -> None: ...
-    def equals(self, value: ComparisonOperand) -> None: ...
-    def not_equals(self, value: ComparisonOperand) -> None: ...
-    def is_in(self, values: ValuesOperand) -> None: ...
-    def is_not_in(self, values: ValuesOperand) -> None: ...
-    def starts_with(self, value: ComparisonOperand) -> None: ...
-    def ends_with(self, value: ComparisonOperand) -> None: ...
-    def contains(self, value: ComparisonOperand) -> None: ...
-    def add(self, value: ComparisonOperand) -> EdgeValuesOperand: ...
-    def subtract(self, value: ComparisonOperand) -> EdgeValuesOperand: ...
-    def multiply(self, value: ComparisonOperand) -> EdgeValuesOperand: ...
-    def divide(self, value: ComparisonOperand) -> EdgeValuesOperand: ...
-    def modulo(self, value: ComparisonOperand) -> EdgeValuesOperand: ...
-    def power(self, value: ComparisonOperand) -> EdgeValuesOperand: ...
-    def round(self) -> EdgeValuesOperand: ...
-    def ceil(self) -> EdgeValuesOperand: ...
-    def floor(self) -> EdgeValuesOperand: ...
-    def absolute(self) -> EdgeValuesOperand: ...
-    def sqrt(self) -> EdgeValuesOperand: ...
-    def trim(self) -> EdgeValuesOperand: ...
-    def trim_start(self) -> EdgeValuesOperand: ...
-    def trim_end(self) -> EdgeValuesOperand: ...
-    def lowercase(self) -> EdgeValuesOperand: ...
-    def uppercase(self) -> EdgeValuesOperand: ...
-    def slice(self, start: int, end: int) -> EdgeValuesOperand: ...
+NodeIndicesOperand: TypeAlias = MultipleAttributesOperand
```

Comment:
Why using a TypeAlias that is equal to a single typing

Reply:
because `NodeIndicesOperand` is exactly the same thing as `MultipleAttributesOperand` and setting it with `TypeAlias` makes it explicit that `NodeIndicesOperand` is just an alias for `MultipleAttributesOperand`